### DISCUSSION TITLE
Sqlite drivers should ignore schema option in entity definitions

### DISCUSSION THREAD

**Discussion by ghost** [mikro-orm/mikro-orm#4449] [General]
**Describe the bug**

I have an application that worked postgres where I have decorators like `@Entity({tableName: 'games', schema: 'someSchema'})` 

I then wanted to make the application work with sqlite, but it generates queries with the schema which fails like so:
```sql
`"select `g0`.* from `shared`.`games` as `g0` where `g0`.`status` = 1 order by `g0`.`started_at` desc limit 1 - no such table: shared.games
```

This also seems to apply to generated migrations. 

I'm using better-sqlite, but it seems like it would happen with the others as well.

**Stack trace**
```
...
```

**To Reproduce**
1. Create a entity with with the schema attribute
2. use a sqlite database

**Expected behavior**
I expect the `schema` attribute to be ignored on databases that don't support them.

**Additional context**

**Versions**

| Dependency | Version |

| - | - |
| node | ? |
| typescript | ? |
| mikro-orm | 5.7.10 |
| your-driver | @mikro-orm/better-sqlite@5.7.10 |

**Comment by B4nan** [comment]
We discuss this on slack already, I disagree the ORM should ignore any explicit options. Or do you know other tools that would behave like that?

I think the solution to your specific problem (that you haven't described here, but IIRC you want to use the same entities for different drivers) is to have metadata discovery hooks so you can modify the metadata dynamically. I will allow doing that in v6.

**Comment by ghost** [comment]
There are alternatives like allowing `@Entity({ postgresql: { schema: 'schemaName'}  })`
